graphql/util.go

```diff
@@ -81,3 +81,38 @@ func IsURL(str string) bool {
 	u, err := url.ParseRequestURI(str)
 	return err == nil && u.Scheme != "" && u.Host != ""
 }
+
+// GetBaseTaskStatusesFromPatchID gets the status of each base build associated with a task.
+// Returns the value in format {buildVariant: {displayName: status}}
+func GetBaseTaskStatusesFromPatchID(r *queryResolver, patchID string) (map[string]map[string]string, error) {
```

Comment:
It would be helpful to create a type alias for the map[string]string part of this, so that it's clear what this is a map of

Reply:
I meant just the map[string]string part. Like this function should return a map[string]SomeStruct